src/sentry/integrations/bitbucket_server/integration.py

```diff
@@ -260,8 +260,7 @@ def get_repositories(self, query=None):
                 for repo in resp.get("values", [])
             ]
 
-        full_query = (query).encode("utf-8")
-        resp = self.get_client().search_repositories(full_query)
+        resp = self.get_client().search_repositories(query)
```

Comment:
Is this because before 3.8 we were not encoding automatically to utf?

Reply:
the `.encode("utf-8")` was a python-2-ism here -- the urlencoding functionalities understand both bytes and text in python 3